### ISSUE TITLE
Should we append another FC layer or dropout layer after the sequence outputs?

### ISSUE THREAD

**Issue by sunshine9409** [microsoft/unilm#261]
Hello~~I am curious about whether  we should append another FC layers or dropout layer after the sequence outputs of bert for the layoutLM pre-training?

**Comment by wolfshow** [comment]
@sunshine9409 Can you please elaborate that in details?

**Comment by sunshine9409** [comment]
> @sunshine9409 Can you please elaborate that in details?

That is, what does the head (sub-net) look like for the pretraining?
